src/ToolBox/superpmi/superpmi/parallelsuperpmi.cpp

```diff
@@ -537,6 +537,13 @@ int doParallelSuperPMI(CommandLine::Options& o)
         if (hStdOutput[i] == INVALID_HANDLE_VALUE)
         {
             LogError("Unable to open '%s'. GetLastError()=%u", arrStdOutputPath[i], GetLastError());
+            delete[] hProcesses;
```

Comment:
All these deletes are not great; some of the array elements contain valid handles that need to be closed before the handles are deleted.

It doesn't much matter if these are deleted/closed before returning, since the process is going to go away immediately after returning. But if we do delete the arrays, we need to make sure we properly close the handles first. Doing that properly (without introducing holders) is probably more complex than it's worth.